### ISSUE TITLE
Change get_user_related_models permissions to also work if the target has a meeting-level scope

### ISSUE THREAD

**Issue by luisa-beerboom** [OpenSlides/openslides-backend#1531]
Just as a meeting admin should be able to delete a user who is only part of that meeting (and the corresponding committee), he should also be able to first call the `get_user_related_models`-presenter and get a sensible preview of what the deletion would cause **and not an error** as is currently the case.

This is important for OpenSlides/openslides-client#1775

**Comment by reiterl** [comment]
I don't understand the issue. I won't fix it.

**Comment by luisa-beerboom** [comment]
@reiterl the `get_user_related_models`-presenter is supposed to be called by the client to get a preview before deleting a user. The preview is supposed to show every way in which the user is important (where is he an admin, where has he done something significant that may stop someone from wanting to delete him, etc).

The purpose of this is to ensure that the deleting user knows what he is doing.

For this reason it does not make sense if there are users that do not have permission to call this presenter _for every user that they are supposed to be able to delete._ 

Meeting admins are supposed to be able to delete every user within their meeting who is not in any other meetings and committees. However the presenter is currently returning missing permission errors every time a meeting admin tries to delete a user within that meeting, _even if he should be allowed to delete him._ (I believe he does this because he checks the CML for every committee (which a pure meeting admin wouldnt have))

So in short: The `get_user_related_models`-presenter should be restricted in the same way as user.delete to ensure that everyone who can call user.delete for a given user can also get a preview for that deletion.

**Comment by jsangmeister** [comment]
@luisa-beerboom what about users with a scope other than `meeting`? They cannot be deleted by a meeting admin. Is the presenter called for them as well or can we ignore them?

**Comment by luisa-beerboom** [comment]
With meeting admins it's only called for targets with a meeting scope, so you can continue to throw errors for them (as well as for any user the meeting admin shouldn't be able to delete, even if he does have a meeting scope)
